docs/src/quick_start.md

```diff
@@ -47,20 +51,42 @@ heatmap(ζ, axis=(; aspect=1))
 
 Fine, we'll re-run this code on the GPU then:
```

Comment:
seems you added a tracer? perhaps mention that?

```suggestion
Fine, we'll re-run this code on the GPU and add a tracer initially concentrated at the center of the domain.
```

Reply:
Oh I see you added these new details at the bottom. I think if reads more natural if they are spelled out at first before people see the code. Otherwise they might think (initially) "oh it's more involved to run on GPU"

Reply:
hmm yes, we wouldn't want them to think that!

Reply:
i'll push an update once the docs build and we can see that things worked (this is the first time we use `@example` with GPU on tartarus I think)